IdentityCore/src/cache/accessor/MSIDLegacyTokenCacheAccessor.m

```diff
@@ -738,7 +741,7 @@ - (MSIDBaseToken *)getTokenByLegacyUserId:(NSString *)legacyUserId
 
     for (NSURL *alias in aliases)
     {
-        MSID_LOG_VERBOSE(context, @"(Legacy accessor) Looking for token with alias %@, clientId %@, resource %@", alias, clientId, resource);
+        MSID_LOG_VERBOSE(context, @"(Legacy accessor) Looking for token with alias %@, clientId %@, resource %@, hashed legacy userid %@", alias, clientId, resource, [legacyUserId msidTokenHash]);
```

Comment:
You can call [legacyUserId msidTokenHash] once on line 741 before going into the aliases loop. It should be the same for all aliases.

Reply:
addressed